sdk/src/cameras/ad-fxtof1-ebz/camera_fxtof1.cpp

```diff
@@ -332,6 +329,7 @@ aditof::Status CameraFxTof1::setFrameType(const std::string &frameType) {
             return status;
         }
         m_details.frameType = *frameDetailsIt;
+        //Turn of the streaming of data
```

Comment:
off (typo) or should it be on?

Reply:
It should be on. Sorry, i will fix this.